jenkins/bootstrap_test.py

```diff
@@ -1980,6 +1980,8 @@ def testValidJobEnvs(self):
         with open(bootstrap.test_infra('jobs/config.json')) as fp:
             config = json.loads(fp.read())
             for job in config:
+                self.assertTrue('sigOwners' in config[job], job)
+                self.assertTrue(type(config[job]['sigOwners']) == list)
```

Comment:
If you and I are the only people touching this file that would be fine.

Since we are composed of a large number of people and "sigOwners": ["UNCLAIMED-PLEASE-SET-THIS-FOR-YOUR-NEW-JOB-OR-YOUR-PRESUBMIT-WILL-FAIL"] makes obvious what needs to happen in a way that `"sigOwners": []` does not.

Also, `"sigOwners": []` is easier to type than `"sigOwners": ["node"]` and developers are lazy. I want to make it easier and more beautiful to create a job that has a defined set of owners.